### PR TITLE
Replace bash with github action in gitops tutorial

### PR DESCRIPTION
**Pull request by kalantar**
Signed-off-by: Michael Kalantar <kalantar@us.ibm.com>
Replace bash script with github action in istio/gitops tutorial.

### CODE REVIEW

**Comment by huang195** [comment]
Looks good, please include doc changes before we merge

File: samples/istio/gitops/templates/experiment.yaml
Comment (huang195) [inline samples/istio/gitops/templates/experiment.yaml]: we don't want to hard code this URL

Comment (kalantar) [reply samples/istio/gitops/templates/experiment.yaml]: I've replaced `iter8-tools` with `MY_ORG` so that it is replaced as part of the tutorial. There is no easy way to do otherwise without a script. Either script so can do git clone and local reference OR add to GH workflow but then need kubernetes credentials. This needs to be addressed in some way first.

File: .github/workflows/gitops-ci.yaml
Comment (huang195) [inline .github/workflows/gitops-ci.yaml]: Do we want to keep these debugging message?

**Comment by huang195** [comment]
Other than the minor nit, everything looks good. Feel free to merge or make changes.

**Review by huang195 — DISMISSED** [review]

**Comment by kalantar** [comment]
@huang195 this is will need to be re-approved after removing the cat statements.
@sriumcp I don't seem to have permission to merge code in this project.

**Review by huang195 — APPROVED** [review]

**Pull request merged**
